src/css/skins/seven.less

```diff
@@ -50,14 +50,16 @@
         -moz-osx-font-smoothing: grayscale;
     }
 
-    .jw-slider-container {
-        .jw-knob {
-            display: none;
-        }
-
-        &:hover {
+    &:not(.jw-flag-touch) {
```

Comment:
This increases specificity, which should be avoided.

If the goal is to override existing ruled on touch devices, add a new `.jw-flag-touch` that overrides the existing rule.